testing/agent-exporter/src/main/java/io/opentelemetry/javaagent/testing/instrumentation/DenyUnsafeInstrumentationModule.java

```diff
@@ -32,10 +31,4 @@ public List<TypeInstrumentation> typeInstrumentations() {
         new ServerInstrumentation(),
         new DefaultStreamMessageInstrumentation());
   }
-
-  @Override
```

Comment:
Where did this method go?

Reply:
based on the comment, I don't think needed anymore with this PR:

> using a system property here will enable the instrumentation when declarative config is used

Reply:
We could use one of the customizers, e.g. add "false" to default properties